src/calendarimportmodel.cpp

```diff
@@ -158,7 +158,7 @@ QVariant CalendarImportModel::data(const QModelIndex &index, int role) const
 
 bool CalendarImportModel::importToNotebook(const QString &notebookUid, bool discardInvitation) const
 {
-    for (const KCalendarCore::Incidence::Ptr &incidence : mEventList) {
+    for (const QSharedPointer<KCalendarCore::Event>& incidence : mEventList) {
```

Comment:
It can be written as `KCalendarCore::Event::Ptr&` since `mEventList` is a `KCalendarCore::Event::List` (which is internally a `QVector<KCalendarCore::Event::Ptr>`. It's more idiomatic than writing the `QSharedPointer<>`.

Reply:
Good suggestions!  Done.